### ISSUE TITLE
Axios upgrade

### ISSUE THREAD

**Issue by Ferdi** [mitodl/micromasters#4516]

**Comment by Ferdi** [comment]
➤ Peter Pinch commented:

Is this upgrade blocked by Searchkit?

**Comment by Ferdi** [comment]
➤ Anna Gavrilman commented:

pdpinch yes, searchkit depends on it

**Comment by Ferdi** [comment]
➤ Peter Pinch commented:

I think you've tried this, but I have to ask: can you upgrade Axios underneath searchkit?

**Comment by pdpinch** [comment]
duplicate of #4383
